api/analytics/analytic_report.py

```diff
@@ -0,0 +1,100 @@
+from flask import jsonify
+import pandas as pd
+
+from helpers.calendar.analytics import RoomAnalytics
+from .write_files import WriteFile
+
+
+class AnalyticsReport():
+    """Get room analytics to export
+       :methods
+           get_dataframe
+           get_all_rooms_summary
+           get_least_used_rooms
+    """
+
+    def get_dataframe(self, rooms_available, start_time, end_time):
+        """ Get all rooms data in a dataframe
+         :params
+            - rooms_available
+            - start_time, end_time
+        """
+        res = []
```

Comment:
Avoid abbreviated variable names, just use response and not res, as abbreviations cause variables to lose meanings